src/components/documentation-card/index.tsx

```diff
@@ -1,28 +1,25 @@
 import Link from 'next/link'
 import { Box, Flex, Text } from '@vtex/brand-ui'
 
-import type { IconComponent } from 'utils/typings/types'
-
 import styles from './styles'
 import { cardContainer, cardTitle, titleContainer } from './functions'
+import { DataElement } from 'utils/typings/types'
 
-export interface CardProps {
+export interface DocumentProps extends DataElement {
   title: string
-  description: string
-  to: string
-  Icon: IconComponent
+}
+export interface CardProps extends DocumentProps {
   containerType: string
```

Comment:
What do you think about enforcing which container types may be provided as props? So that we can avoid typos while developing

Reply:
Good! I will add it